packages/react-charting/src/components/VerticalBarChart/__snapshots__/VerticalBarChart.test.tsx.snap

```diff
@@ -2672,9 +2607,6 @@ exports[`VerticalBarChart snapShot testing renders enabledLegendsWrapLines corre
       </div>
     </div>
   </div>
-  <span
```

Comment:
why have few elements been removed from the snapshot

Reply:
I am not sure , but should not cause any issues as this is an empty tag.
![image](https://github.com/microsoft/fluentui/assets/103660888/86f1c42c-1774-4984-a367-927f5e9ace4d)